spec/src/main/asciidoc/changelog.adoc

```diff
@@ -27,6 +27,7 @@ Changes marked with icon:bolt[role="red"] are breaking changes relative to previ
 ** Added ProcessCpuTime as a new optional base metric.
 
 * Changes in 2.1
+** Added ProcessCpuTime as a new optional base metric.
```

Comment:
That will happen in 2.1.1 AFAIK (not 2.1), so perhaps we should only add this to the changelog in the 2.1.x branch? (I mean we would add a new "Changes in 2.1.1" section, but just in the 2.1.x branch, don't mention it in master) The changelog will look a bit weird if we list the **same** change in sections for 2.0, 2.1 and 2.2.

Reply:
I agree it seems a bit weird to have the same thing added to all 3, but I think we need to be clear about when it was introduced.  How about we say it like this...?

* Changes in 2.2
** Added ProcessCpuTime as a new optional base metric

* Changes in 2.1
** (2.1.1) Added ProcessCpuTime as a new optional base metric

* Changes in 2.0
** (2.0.3) Added ProcessCpuTime as a new optional base metric.

Reply:
That sounds good